templates/less/components/form.less

```diff
@@ -8,62 +8,100 @@
   }
   @media (min-width : 451px) {
     width: 400px;
+    height: 250px;
   }
   border-radius: 7px;
   padding-top: 25px;
   margin-bottom: 30px;
   background-color: white;
 
-  input {
-    width: 100%;
-    font-size: 16px;
-    font-weight: 200;
-    height: 55px;
-    padding-left: 12.5%;
-    margin: 0 auto;
+
+  input[disabled] {
+    background-color: transparent;
+    cursor: default !important;
```

Comment:
also, as an `input`, pretty sure this needs to go inside of `label` like the other inputs.